libs/execution_context/ebpf_native.c

```diff
@@ -11,6 +11,8 @@
 #define DEFAULT_PIN_ROOT_PATH "/ebpf/global"
 #define EBPF_MAX_PIN_PATH_LENGTH 256
 
+static const uint32_t _ebpf_native_marker = 'enat';
```

Comment:
```suggestion
static const uint32_t _ebpf_native_marker = 'entv';
```
just a suggestion to avoid confusion with NAT